plugin/src/sbt-test/scalafmt-sbt/sbt/test

```diff
@@ -132,6 +132,8 @@ $ copy-file changes/bad.scala p17/src/main/scala/Test2.scala
 -> p17/scalafmtCheck
 # prevent reading the source (without changing the mtime) to detect actual/uncached invocation of scalafmt
 $ exec chmod 000 p17/src/main/scala/Test.scala
+######## incremetnal checking should carry over failure without actually processing the file
```

Comment:
if we *really* want to assert that the file was not processed, we would have to check that no exception was raised (AFAIK the statement below doesn't differentiate the permission denied caused by the `chmod` from a `false` return value)

Reply:
sorry I was confused, `scalafmtCheck` raises an exception no matter what (false is never returned), so we would have to assert on the type of exception